pom.xml

```diff
@@ -40,15 +40,15 @@
     <changelist>-SNAPSHOT</changelist>
 
     <!-- jenkins versions -->
-    <jenkins.version>2.332.4</jenkins.version>
-    <bom.artifactId>bom-2.332.x</bom.artifactId>
-    <bom.version>1763.v092b_8980a_f5e</bom.version>
+    <jenkins.version>2.401.1</jenkins.version>
+    <bom.artifactId>bom-2.401.x</bom.artifactId>
+    <bom.version>2244.vd60654536b_96</bom.version>
 
     <!-- dependency versions -->
     <authentication-tokens.version>1.3</authentication-tokens.version>
     <docker-commons.version>1.15</docker-commons.version>
     <google-oauth-plugin.version>0.8</google-oauth-plugin.version>
-    <kubernetes-client-api.version>6.3.1-206.v76d3b_6b_14db_b</kubernetes-client-api.version>
+    <kubernetes-client-api.version>6.8.1-220.v1f66736f9a_42</kubernetes-client-api.version>
```

Comment:
This version number will need to be checked when this PR is merged https://github.com/jenkinsci/kubernetes-client-api-plugin/pull/221 and we have a new release version for kubernetes-client-api. It needs to be at least 6.8.0 to be compatible with the snakeyaml 2.0 changes.

Reply:
https://github.com/jenkinsci/kubernetes-client-api-plugin/pull/221